client/src/app/(main)/explore/page.tsx

```diff
@@ -1,11 +1,60 @@
-import { OpportunityCard } from "@/ui/explore/OpportunityCard"
+"use client";
+import { Typography } from "@mui/material";
+import "@/app/(main)/explore/explore.css";
+import { OpportunityCard } from "@/ui/explore/OpportunityCard";
 
 export default function Page() {
-  return (
-
 
-    <OpportunityCard/>
+  return (
+    <div className="page-container">
```

Comment:
Unnecessary divs. Since this page is pretty much a two column layout, why not just do make the outmost div be a grid, and do gridTemplateColumns: {some percentage}% {some percentage}%?. First column being a scrollable for the explore cards, and the second column for the filter section.